staging/src/k8s.io/kubectl/pkg/describe/describe.go

```diff
@@ -3040,10 +3040,10 @@ func (d *NodeDescriber) Describe(namespace, name string, describerSettings Descr
 
 	lease, err := d.CoordinationV1().Leases(corev1.NamespaceNodeLease).Get(context.TODO(), name, metav1.GetOptions{})
 	if err != nil {
-		if !errors.IsNotFound(err) {
+		if !errors.IsNotFound(err) || !errors.IsForbidden(err) {
 			return "", err
 		}
-		// Corresponding Lease object doesn't exist - print it accordingly.
+		// Corresponding Lease object doesn't exist or no access to it - print it accordingly.
 		lease = nil
```

Comment:
prefer printing something about not being able to access it instead of "unset"

Reply:
Does "<hidden>" look good?

Reply:
plumb the lease retrieval error down into describeNode, and display something like https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/kubectl/pkg/describe/describe.go#L2109-L2113

Reply:
@liggitt please take a look.